syft/grid/clients/data_centric_fl_client.py

```diff
@@ -284,3 +285,77 @@ def delete_model(self, model_id: str) -> bool:
 
     def __str__(self) -> str:
         return f"<Federated Worker id:{self.id}>"
+
+    @staticmethod
+    def simplify(worker: AbstractWorker, data_centric_fl_client: "DataCentricFLClient") -> tuple:
+
+        # Simplify the attributes for DataCentricFLClient
+        address = pickle.dumps(data_centric_fl_client.address)
+        id = pickle.dumps(data_centric_fl_client.id)
+        is_client_worker = pickle.dumps(data_centric_fl_client.is_client_worker)
+        log_msgs = pickle.dumps(data_centric_fl_client.log_msgs)
+        verbose = pickle.dumps(data_centric_fl_client.verbose)
+        encoding = pickle.dumps(data_centric_fl_client.encoding)
+        timeout = pickle.dumps(data_centric_fl_client.timeout)
+
+        return (address, id, is_client_worker, log_msgs, verbose, encoding, timeout)
+
+    @staticmethod
+    def detail(
+        worker: AbstractWorker, client_tuple: tuple
+    ) -> Union["DataCentricFLClient", int, str]:
+
+        address, id, is_client_worker, log_msgs, verbose, encoding, timeout = client_tuple
+
+        # detail client attributes
+        address = pickle.loads(address)
+        id = pickle.loads(id)
+        is_client_worker = pickle.loads(is_client_worker)
+        log_msgs = pickle.loads(log_msgs)
+        verbose = pickle.loads(verbose)
+        encoding = pickle.loads(encoding)
+        timeout = pickle.loads(timeout)
+
+        # create torch hook
+        hook = sy.TorchHook(torch)
+        me = hook.local_worker
+
+        # if worker with same id exist return that worker 2 worker with same id raises error
+        if id in me._known_workers.keys():
+            return me._known_workers[id]
+
+        Client = DataCentricFLClient(
+            hook, address, id, is_client_worker, log_msgs, verbose, encoding, timeout
+        )
+
+        return Client
+
+    @staticmethod
+    def get_msgpack_code() -> Dict[str, int]:
```

Comment:
Q: Shouldn't we have a ```syft-proto``` for this?

Reply:
I will add the code in syft-proto, but for now, this would work also it's easier to use this function if add any new class which need to be serialised, we don't need to wait fr adding it to syft-proto.

Reply:
Could you create an issue in ```PySyft``` with good first issue regarding this.

Reply:
okay